modules/ComscoreStreamingTag/ComscoreStreamingTag.manifest.php

```diff
@@ -0,0 +1,72 @@
+<?php 
+/**
+ * The ComScoreStreamingTag plugin manifest
+ */
+
+return array(
+	'ComScoreStreamingTag' => array(
+		'description' => "Supports sending Streaming tag player analytics events to comScore.",
+		'label' => "comScoreStreamingTag",
+		'attributes'=> array(
+			'c2' => array(
+				'doc' => "Used to specify the comScore client ID (also called 'Customer C2') which identifies the comScore account for which the data should be collected. Expects a number with 7 or 8 digit.",
+				'label' => 'comScore client ID',
+				'type' => 'String',
+			),
+
+			'persistentLabels' => array(
+				'doc' => "Advanced configuration setting! Used to specify 'persistent labels' which are added to all measurements. Expects a comma-separated list of 'name=value' assignments. Please do not use this setting unless you are explicitly instructed to do so by your comScore client account representative.",
+				'label' => "Persistent labels",
+				'type' => 'String',
+			),
+
+			'labelMapping' => array(
+				'doc' => "Used to map media metadata values to specific labels. The mapping is applied to the collected data whenever the player loads new media. Expects a comma-separated list of 'label=mapped-source' assignments. The 'mapped-source' can be a dynamic value - i.e., pointing to media metadata fields - or a static value (needs to be enclosed in quote characters). Please contact your comScore client account representative for more instructions on proper use of this configuration setting.",

```

Comment:
I don't think this documentation was updated per the shift to normal usage of {mediaProxy.entry.name} style substitutions.